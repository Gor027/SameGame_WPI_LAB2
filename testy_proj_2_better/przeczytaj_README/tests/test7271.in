 3Y33Y  333Y3 Y YYY 
YYYY  Y3 33Y3 3 33Y  Y3 YY 3Y3YY YYY
Y 3333   Y3Y3YY33 333 3   3   333 Y3Y 3 Y YYYY33Y YY 3Y Y33  Y3333   3Y
Y3YY3  3 3 3Y  YY33 Y Y Y 3 
 YY YY3   333 YY 3YY Y33YY3 33YY YY 33YYY 333 3Y  3 3 33Y 333   3  YYY
Y 3  YYYY333Y3  33YYYY3Y Y33Y3 Y Y   3Y Y3
YYY 33Y3 3Y3   YYY YY    3   3 Y3  Y3YYYYY3 Y3YY3Y33 YY3 Y YY Y   3YY YYY3YY 3
3Y33Y3 3Y Y3 YY3 YY Y YYY YY33Y 3 Y3Y  3YY3Y3  Y  YY3YYY  3YY
3 Y 333Y  Y3Y3Y33YY3Y33 33  YYY  3333Y33 3 3 Y 33Y 33
3 333 3  3YY3 YYY 3 3YYYY3 Y 3YY  333Y3YY33  3
