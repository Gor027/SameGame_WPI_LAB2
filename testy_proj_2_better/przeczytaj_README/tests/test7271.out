  Y        Y
  Y       3Y      Y
  3 3     3Y3   YYY   3
  Y 3Y   3333 Y3333 3 3      3
 3Y333Y3 3333 3Y333 Y Y YY   3YY  YYY 3 Y
YY3YYY33 Y33Y YYY3Y3YY3 YY3 Y333  Y333YYY3Y    3
Y3Y33YYY 3Y33YYY33YYYY33YY3333YY YYY33YYYY3YYY Y  YY  Y
YY3YY3Y333Y3Y3Y3YYYYYYYYY333Y3YY3YY3YYY3YY33333YY 333333  333
YYY33333Y3Y3YYY33YYYY33Y333Y3YYY3Y333Y33Y33YYY3Y3Y3Y3YY3Y33YY 3 3  YY3Y
YY33333YY3YY33YYYY333YYYY3YYY3YY33333Y3YY33333Y33YY33YYYYY3YYYY3333YYYYYY3YY3
